src/neptune_optuna/impl/__init__.py

```diff
@@ -415,7 +419,8 @@ def log_study_metadata(
         _log_trials(run, study, study.trials, namespaces=namespaces)
 
     if log_distributions:
-        run["study/distributions"].append([stringify_unsupported(trial.distributions) for trial in study.trials])
+        for i, trial in enumerate(study.trials):
+            run[f"study/distributions/trial/{i}"].append(stringify_unsupported(trial.distributions))
```

Comment:
```suggestion
            run[f"study/distributions/trial/{i}"] = stringify_unsupported(trial.distributions)
```
Distributions are not a series